### DISCUSSION TITLE
Getting the output of PIR sensors using "Virtual on/off device" in NHC

### DISCUSSION THREAD

**Discussion by BalderikTheSecond** [joleys/niko-home-control-II#191] [Q&A]
I read in the readme about the possibility to the output of the PIR sensor in Home Assistant, but it seems I am doing something wrong.

I created the virtual device and linked it to the routine of the PIR sensor. 

![Screenshot 2024-11-30 154654](https://github.com/user-attachments/assets/b4d420cc-792e-43a8-8ab3-e02d2c4c70a3)

Loaded my new configuration successfully in NHC2.
Waited a while and then restarted Home Assistant.

However, the new virtual device does not appear in Home Assistant.

What could I be doing wrong?

**Comment by BalderikTheSecond** [comment]
In the meantime, the debugging enabled and restarted HA but the virtual devices are also not found in the log file.

2024-12-02 22:22:00.611 INFO (MainThread) [custom_components.nhc2.switch] → Found 0 NHC Virtual flags

**Comment by tijsverkoyen** [comment]
Have you included them to be exposed to the app?

**Comment by BalderikTheSecond** [comment]
Forgive my ignorance, but how should I expose them to the app?

When I look into the profiles of the mobile apps and touch screens in the Niko dashboard, all available devices are selected, but it does no show any flags (aka virtual flags) that I can select (or deselect).

**Comment by BalderikTheSecond** [comment]
I think I found it. I went into the Niko app and then flags into buttons in the app, and then they appear in HA.
